Database/Corrections/QuestieObjectFixes.lua

```diff
@@ -13,6 +13,7 @@ function QuestieObjectFixes:Load()
         [148516] = {"Tablet of Beth'Amara",nil,nil,{[16]={{34.1,50.5},{34.3,55.2},{35.19,58.02},{36,57.6},{36.8,60.5},{37.07,51.86},{38.4,60.3},{38.91,53.46},{39.54,64.32},{39.6,48.14},},},16,},
         [177241] = {"Araj's Phylactery",nil,nil,{[28]={{45.27,69.21},},},28,},
         [179908] = {"Slagtree's Lost Tools",nil,nil,{[47]={{53.3,38.8},{57.4,42.6},{57.5,42.6},{66.4,44.8},{71,48.6},{72.6,52.9},},},47,},
+        [153123] = {"Kim'jael's Equipment",nil,nil,{[16]={{56.1,30.1},{56.3,28.8},{56.4,28.4},{56.5,28.8},{57.0,28.4},{57.0,29.9},{57.9,29.1},{58.2,27.7},{58.4,25.0},{58.5,25.0},{58.6,26.0},{58.6,29.1},{58.9,28.4},{59.2,23.0},{59.6,30.9},},},16,},
```

Comment:
It would be better to use the `spawns` key like other fixes below already do, instead of overwriting the whole object.
```suggestion
        [153123] = {
            [QuestieDB.objectKeys.spawns] = {[16]={{56.1,30.1},{56.3,28.8},{56.4,28.4},{56.5,28.8},{57.0,28.4},{57.0,29.9},{57.9,29.1},{58.2,27.7},{58.4,25.0},{58.5,25.0},{58.6,26.0},{58.6,29.1},{58.9,28.4},{59.2,23.0},{59.6,30.9},},},
```

Reply:
Done. Thanks for the feedback!